project/settings/prod.py

```diff
@@ -39,3 +39,5 @@
 
 # use mailgun's REST API
 ALLOW_REQUESTS_TO_MAILGUN = True
+
+DIVERT_REMOTE_CONNECTIONS = os.environ.get('DIVERT_REMOTE_CONNECTIONS')
```

Comment:
should this be in `settings/environment.py` where all the other env var settings are pulled in?

Reply:
no because I only want this to apply to production and staging.  Demo, development, and everywhere else can just have the hardcoded True